### ISSUE TITLE
Can't support object hashes

### ISSUE THREAD

**Issue by Molunerfinn** [ThisIsManta/stylus-supremacy#5]
I find a interesting thing:
```stylus
$obj = {
  color: #000
}

.title
  color: $obj.color // -> got error
```
while the situation below is ok:
```stylus
$obj = {
  color: #000
}

.title
  color: $obj['color'] // -> OK
```
Is it a bug?

Thx for u hard work! It's perfect.

**Comment by ThisIsManta** [comment]
Hello,

Thank you for reporting this. Please spare me a day to check if this is really a bug.

Cheers!

**Comment by Molunerfinn** [comment]
: )~

**Comment by ThisIsManta** [comment]
I can confirm this is a bug. Fixing in progress.

**Comment by ThisIsManta** [comment]
@Molunerfinn, this issue has been resolved in version [1.6.3](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v1.6.3).

If you are using **Atom** package (extension), please wait few more days because I do not have authorization to publish a new version.

Thank you for making **Stylus Supremacy** a better library.

**Comment by Molunerfinn** [comment]
Amazing~! Actually I use VSCode haha.
Thank you for your hard work, it's a wonderful tool to me :)
